timezone_converter/comparison_view.py

```diff
@@ -55,7 +55,14 @@ def _build_table(self) -> Table:
         fmt = '%Y-%m-%d %H:%M'
 
         current_hour = datetime.now().hour
-        for hour in range(24):
+        if single_hour is None:
+            hour_range = range(24)
+        elif single_hour == 'now':
+            hour_range = range(current_hour, current_hour + 1)
+        else:
+            hour_range = range(int(single_hour), int(single_hour) + 1)
```

Comment:
I prefer to use something like we discussed in #15, a default value that will later change. I also prefer that the argument is just called `hour` (and that we provide it as an instance argument, as I mentioned in `main.py`). This is how this part would end up looking like:

```py
        hour_range = range(24)
        if self.hour is not None:
            hour_range = range(self.hour, self.hour + 1)
```

As I mentioned before, we would not use `now` and the current hour will be provided by `const` when no value is provided to `--single`.

We are also converting the value to `int` so no need to do it here.